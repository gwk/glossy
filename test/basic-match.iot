{
  'args': ['test/basic.legs', '-match',
    'a',
    'bd',
    'bcd',
    'eg',
    'efg',
    'effg',
    'h',
    'hh',
    'ij',
    'k',
    'l',
    '',
  ],
  'links': {'test': 'test'},
}