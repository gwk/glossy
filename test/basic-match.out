match: ' ' -> s
match: 'a' -> a
match: 'bd' -> bc_d
match: 'bcd' -> bc_d
match: 'eg' -> ef_g
match: 'efg' -> ef_g
match: 'effg' -> ef_g
match: 'h' -> h_
match: 'hh' -> h_
match: 'ij' -> ij
match: 'k' -> k_l
match: 'l' -> k_l
match: '' -- invalid
match: 'aa' -- invalid
match: 'ab' -- invalid
match: 'b' -- invalid
match: 'bb' -- invalid
match: 'bccd' -- invalid
match: 'e' -- invalid
match: 'ef' -- invalid
match: 'g' -- invalid
match: 'ah' -- invalid
match: 'ha' -- invalid
match: 'i' -- invalid
match: 'j' -- invalid
match: 'ija' -- invalid
match: 'kl' -- invalid
